php-laravel-phan.md

```diff
@@ -0,0 +1,150 @@
+# Introduction
```

Comment:
Do we also give recommendations regarding IDE etc?

Reply:
For IDE we thought to keep a different documentation.
For each technology which IDE to use, what extensions to install and workspace configuration settings to be used.
And any other related information in that.